### DISCUSSION TITLE
0.14.0 + ha proxy

### DISCUSSION THREAD

**Discussion by danuals** [blakeblackshear/frigate#9677] [General]
I use frigate proxy with 13.1 fine. frigate is on my synology, ha on a nuc. 
with 14 dev when opening frigate proxy in ha it loads with 404.
dashboard loads, I see notifications and can load clips etc. but click on a camera from dash and its 404 page not found

**Comment by NickM-27** [comment]
0.14 is not ready for users to use

**Reply by danuals** [reply]
cool that's why I did this here not as an issue, just a general query, ill go back to 13.1. ps love the new ui

**Comment by jriffe6** [comment]
I am getting sort of the same thing with the new version.  I got the latest docker image and am using HA proxy and when i click on it in the sidebar it comes up with a 404 but if i click on the dashboard link it loads but only if i go into the app and click on the link there.  Also when i click on one of the cameras it just goes to a black page and never loads so I went back to the 0.12 version and that is working fine.  I used the same docker compose file as the 0.12 version just changed the location of the image and the change for the config.yml section

**Reply by zee-shany** [reply]
Anyone able to make 0.14 work with frigate proxy 1.5? 
i'm getting 502 bad gateway if i try to open webUI from the frigate proxy add-on.
on browser when entering frigate IP http://10.x.x.x:5000 it says "ERR_CONNECTION_REFUSED".
any help is appreciated. I've been searching since two days to find any fix.
